invenio_communities/templates/semantic-ui/invenio_communities/details/subcommunity/index.html

```diff
@@ -14,20 +14,34 @@
 {%- set title = community.metadata.title + _(" subcommunities") -%}
 
 {%- block javascript %}
-    {{ super() }}
-    {{ webpack['invenio-communities-subcommunities.js'] }}
+{{ super() }}
+{{ webpack['invenio-communities-subcommunities.js'] }}
 {%- endblock %}
 
 {%- block page_body %}
 {{ super() }}
-<div class="ui container fluid rel-mt-3 rel-mb-2 compact ml-0-mobile mr-0-mobile">
-  <div class="ui container community-search page-subheader flex align-items-center justify-space-between">
-    <h1 class="ui large header m-0">{{_("Subcommunities")}}</h1>
+<div class="ui container fluid page-subheader rel p-10 mr-0-mobile ml-0-mobile">
+  <div class="collection-header ui container align-items-center justify-space-between">
+    <h1 class="ui medium header m-0">{{ config.get("APP_RDM_SUBCOMMUNITIES_LABEL", _("Subcommunities")) }}</h1>
+  </div>
+</div>
+<!-- Breadcrumbs -->
+<div class="ui container mb-0">
+  <div class="ui breadcrumb">
+    <a class="section"
+      href="{{ url_for('invenio_app_rdm_communities.communities_browse', pid_value=community.slug ) }}">
```

Comment:
minor: I guess we're aware of the "backwards" reference to Invenio-App-RDM? In theory "Browse" as a concept only exists there.

Reply:
Yes, this was discussed when we first implemented the browse page. Unfortunately, there's some entanglement that IMO should be cleaned up when we have more time.